### ISSUE TITLE
fish crash when using autocompletion

### ISSUE THREAD

**Issue by Rudloff** [termux/termux-packages#1845]
![screenshot_20171127-101513](https://user-images.githubusercontent.com/840125/33259168-512fba4c-d35c-11e7-9798-f920ba12992f.png)

Here is how I can reproduce the crash:
1. Open a fish shell.
1. Type `fish`.
1. Press tabs twice.

However I can't reproduce when connect from my comptuer via SSH or ADB.
Here is my `termux-info`:
```
Updatable packages:
All packages up to date
System information:
Linux localhost 3.4.0-g64b352a #1 SMP PREEMPT Thu Oct 26 18:26:41 BST 2017 armv7l Android
Termux-packages arch:
arm
Android version:
6.0.1
Device manufacturer:
Fairphone
Device model:
FP2
```

**Comment by fornwall** [comment]
Thanks - follow the existing issue #909 for progress!

**Comment by fornwall** [comment]
This seems to be fixed in version 2.7.0 of fish - update with `pkg up`!
